orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/RollingRedBlackStrategySpec.groovy

```diff
@@ -52,15 +55,20 @@ class RollingRedBlackStrategySpec extends Specification {
       targetHealthyDeployPercentage: 90
     ]
 
-    def stage = new Stage(Execution.newPipeline("orca"), "whatever", ctx)
+    def pipeline = Execution.newPipeline("orca")
+
+    def createAsgStage = new Stage(pipeline, "createServerGroup", ctxCreateASG)
+    def stage = new Stage(pipeline, "whatever", ctx)
+
+    pipeline.stages << createAsgStage << stage
```

Comment:
for what it's worth the `ExecutionBuilder` makes building a pipeline graph slightly nicer.

Reply:
i was trying to do minimal changes (since the plan is to totally re-write this ut... one day)... going to fix it up since i tweaked some more code anyway